plugins/jobs/server/__init__.py

```diff
@@ -42,7 +46,18 @@ def scheduleLocal(event):
         fn = getattr(module, job.get('function', 'run'))
         fn(job)
 
+@access.token(scope=constants.REST_CREATE_JOB_TOKEN_SCOPE)
+def _authorizeRestJobCreation(event):
+    tokenModel = ModelImporter.model('token')
+    token = getCurrentToken()
+
+    if not tokenModel.hasScope(token, constants.REST_CREATE_JOB_TOKEN_SCOPE):
+        raise AccessException(
+            'Invalid token scope.\n'
+            'Required: %s.\n' % (constants.REST_CREATE_JOB_TOKEN_SCOPE))
```

Comment:
Would this also help with the recent issue we found where invalidly scoped tokens where throwing 401s and saying "not logged in" when really it was the case that the token didn't have the correct scope?  If we can improve that error handling as a part of this PR that would be rad.